pep-0548.rst

```diff
@@ -0,0 +1,338 @@
+  gain compatibility with platforms that only provide Python 3.x, should
+  specifically target ``python2`` (as this is available by default almost
+  everywhere that Python 2 is available, and is easily added as a symlink if
+  it is missing).
+* standalone scripts that want to support both older platforms where ``python``
+  refers to ``python2`` *and* newer platforms or virtual environments where it
+  refers to ``python3`` should be updated to run in the common subset of Python
+  2.7 & 3.6+ using either the `regular porting guide`_ in the Python reference
+  documentation or else the `conservative porting guide`_ developed as part of
+  Fedora's migration to using Python 3 as the system Python runtime.
+* standalone scripts designed to run under both Python 2 and 3 may reasonably
+  require the availability of the ``six`` compatibility library. However,
+  avoiding this runtime dependency will improve compatibility with older
+  platforms that don't provide that component by default.
+* These distinctions should be made in shebang lines for Python scripts, when
+  invoking Python from shell scripts, when invoking Python via the C
+  ``system()`` call, or when invoking Python in any other context.
+* When reinvoking the interpreter from a Python script, querying
+  ``sys.executable`` to avoid hardcoded assumptions regarding the
+  interpreter location remains the preferred approach.
+
+.. _regular porting guide: https://docs.python.org/3/howto/pyporting.html
+.. _conservative porting guide: https://portingguide.readthedocs.io/en/latest/process.html
+
+
+Recommendations for Development of Cross-Platform Applications
+==============================================================
+
+* cross-platform applications should use appropriate mechanisms (such as Python
+  virtual environments or system level application containers) to avoid
+  depending on the default version of Python installed on any particular
+  deployment platform.
+
+
+Recommendations for Platform Providers
+======================================
+
+* Unix-like software distributions (including systems like Mac OS X and
+  Cygwin) should install the ``python2`` command into the default path
+  whenever a version of the Python 2 interpreter is installed, and the same
+  for ``python3`` and the Python 3 interpreter.
+* When  invoked, ``python2`` should run some version of the Python 2
+  interpreter, and ``python3`` should run some version of the Python 3
+  interpreter.
+* The more general ``python`` command should be handled using one of the
+  following techniques:
+
+  * if present, always refers to the same target as ``python2``
+    (e.g. Debian 7/8/9, RHEL/CentOS 5/6/7, Ubuntu LTS 14.04 & 16.04)
+  * if present, always refers to the same target as ``python3``
+    (e.g. Arch, OpenMandriva)
+  * if present, refers to the same target as ``python3`` by default, can be
+    switched by a system administrator to refer to the same target as
+    ``python2`` (e.g. Gentoo)
```

Comment:
Technically, I believe Debian *could* support this through its [alternatives system](https://wiki.debian.org/DebianAlternatives), although 1) this isn't currently implemented for Python, and 2) I'm not even sure it would be acceptable to the Debian Python community.